app/Http/Controllers/ParamController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Param;
+use App\Models\Project;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class ParamController extends Controller
+{
+    public function index(Project $project): JsonResponse{
+		return new JsonResponse($project->params);
+	}
+	
+	public function store(Project $project,Request $request): JsonResponse{
+		$param = Param::create([
+			'value' => $request->get('value'),
+			'order' => $request->get('order'),
+			'project_id' => $project->id,
+		]);
```

Comment:
Suggestion: on peut aussi directement utiliser la relation entre project et params 
```suggestion
		$project->params->create([
			'value' => $request->get('value'),
			'order' => $request->get('order'),
		]);
```